gen/epilogue.jl

```diff
@@ -5,10 +5,14 @@ const GDAL_DATA = Ref{String}()
 const PROJ_LIB = Ref{String}()
 
 function __init__()
+    # Cleanup at exit
+    atexit(gdaldestroy)
+    # atexit(() -> gdaldumpopendatasets("/dev/stderr")) # TODO find a way to pass a pointer to stderr
```

Comment:
Didn't check, but could this work? `pointer_from_objref(stderr)`

Reply:
I have tested a few combination with `stdout`, `stderr` and their `struct` members. Everything led to a Julia crash:
- `gdaldumpdatasets(pointer_from_objref(stderr))` partially worked one time but did not print the opened dataset list, just GDALDumpOpenDatasets return result (number of opened datasets: `return static_cast<int>(poAllDatasetMap->size())`)
- `gdaldumpdatasets(pointer_from_objref(stdout))` printed a list of opened dataset before crashing Julia

I propose to clean epilogue.jl and raise an issue on the proper usage of `gdaldumpopendatasets`